### PR TITLE
Add upgrade-insecure-requests to CSP

### LINKED ISSUE #88: netlify: silence "Insecure mixed content detected in"
@mveytsman Could try adding `<meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">` or equivalent to our CSP, might do the trick and/or break things?

### PR DESCRIPTION
**Pull request by mveytsman**
Creating this PR to test whether this closes #88

### CODE REVIEW

**Comment by shazow** [comment]
Nice try T_T

File: nuxt.config.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default {
       { charset: 'utf-8' },
       //{ name: 'viewport', content: 'width=device-width, initial-scale=1' },
       { hid: 'description', name: 'description', content: "The Million Dollar Homepage as an Ethereum Smart Contract and DApp: A glimpse into what the future of web integrated with modern blockchain technology could be like." },
-      process.env.NODE_ENV === 'production' ? { 'http-equiv': "Content-Security-Policy", content: "default-src 'self'; script-src 'self' 'self' *.infura.io; connect-src 'self' *.infura.io *.walletconnect.org wss://*.walletconnect.org; style-src 'self' 'unsafe-inline'; img-src * data:"} : {},
+      process.env.NODE_ENV === 'production' ? { 'http-equiv': "Content-Security-Policy", content: "upgrade-insecure-requests; default-src 'self'; script-src 'self' 'self' *.infura.io; connect-src 'self' *.infura.io *.walletconnect.org wss://*.walletconnect.org; style-src 'self' 'unsafe-inline'; img-src * data:"} : {},
```
Comment (shazow) [inline nuxt.config.js:39]: Netlify doesn't use this, gotta change it in the meta header

**Comment by mveytsman** [comment]
This suppresses the mixed content warning in the browser which might be worth it even if it doesnt fix #88

**Comment by shazow** [comment]
@mveytsman I'm down. Wanna add the equivalent to the dev-environment block above for consistency?

**Comment by shazow** [comment]
Wanna merge this?

**Pull request merged**